### ISSUE TITLE
Can't select text to edit it

### ISSUE THREAD

**Issue by rouviere** [edge-reflow/issues#84]
I have been trying to select text to edit it. I have double clicked many times and while the cursor will show up at the end of the text line and I can delete and replace it, I can't double click to and select a letter or a word to apply different styling to it. This is on Mac if it makes any difference.

**Comment by jtlafferty** [comment]
I believe you have run into the terrible bug that we found just as we had already released our public preview. The bug happens when you have used the scrollbar. After using the scrollbar, you can no longer select text for editing via highlighting. You can use the keyboard to select text. We have now fixed this bug, but, not in our public release. Our apologies. 

If you save your project and restart Reflow, the problem also fixes itself. But, it will return when you use drag the scrollbar up or down.

**Comment by rouviere** [comment]
Hi Joan,
Thanks so much for your personal reply. Our developer and I have been using Edge Reflow all day, and I'm not sure it's quite there yet. Using Lynda.com's training, which is still pretty rudimentary, we did the responsive designs for a fairly simple website in it. It was difficult even with an accomplished CSS/HTML developer to help me, and just at the end, two of the three layouts we made fell apart. So we finally made a screenshot of the one that was hanging together, then undid until we got back to where the other two views worked. We took screenshots of them for the developer to use with the site design's original Illustrator file, and then gave up. 

That having been said, we still found Edge Reflow extremely helpful in being able to visualize how to resize a design for phone, tablet and desktop, and as the product develops, we plan to continue to work with it. We are looking forward to some more of the bugs being worked out and also to better training as well as more feedback and tips on the forums in the future. 

We also found the Edge web fonts in the program very limited especially in the condensed san serif category. 

However, we're extremely grateful for Adobe's continuing efforts to push the envelope to provide solutions for these issues. We love and live in the creative cloud and Adobe applications, and have since the earliest versions of your software. 

Best of luck in your continuing efforts. 

Donna Anderson
Creative Director
